google/cloud/security/iam/inventory/storage.py

```diff
@@ -55,6 +55,7 @@ class InventoryTypeClass(object):
     RESOURCE = 'resource'
     IAM_POLICY = 'iam_policy'
     GCS_POLICY = 'gcs_policy'
+    Supported_TypeClass = [RESOURCE, IAM_POLICY, GCS_POLICY]
```

Comment:
either SUPPORTED_TYPECLASS or supported_typeclass. Camelcase only for class names.